src/QueryLogger.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+namespace Muffin\Webservice;
+
+class QueryLogger extends \Cake\Database\Log\QueryLogger
```

Comment:
I would do debug as the level and send the sql as the string and params/timing as additional context for the message. From what I remember psr3 lets you give context on each message. If I am wrong, json is a good alternative.